packages/wonder-blocks-core/src/hooks/use-on-mount-effect.ts

```diff
@@ -29,7 +29,7 @@ import * as React from "react";
  * If you only need to do something on mount, don't return a cleanup function from `callback`.
  */
 export const useOnMountEffect = (
-    callback: () => undefined | (() => void),
+    callback: () => void | undefined | (() => void),
```

Comment:
This can be `callback: () => void | (() => void)` since a `void` return type essentially ignores the type of the return, see https://www.typescriptlang.org/play?ssl=1&ssc=31&pln=1&pc=1#code/DYUwLgBAZg9jBcEAUBKCBeAfBAbjAlgCYbJpYQCsA3AFBA.